lib/ci/test/github_proxy_test.rb

```diff
@@ -0,0 +1,56 @@
+require "minitest/autorun"
```

Comment:
Maybe I should move the "test" directory to the root level and add this project to Travis. I added tests just to help me out during development, but if we don't run them they'll end up becoming broken and useless.